.travis.yml

```
@@ -2,7 +2,8 @@ sudo: required
 language: bash
 env:
   matrix:
-    - PLATFORM=buildpack
+    - PLATFORM=ballerina
+    - PLATFORM=buildpack    
```

Can you please remove the trailing spaces from line 6?
```suggestion
    - PLATFORM=buildpack
```